operations/network/management_network/mellanox/canu_verify_generate_compare_switch_connfiguration.md

```diff
@@ -0,0 +1,469 @@
+# Use CANU to verify, generate or compare switch configuration
```

Comment:
The new file name contains a excess ''n" (canu_verify_generate_compare_switch_con**n**figuration.md), so it is treated as a new file, instead of renaming.

Reply:
Fixed and removed duplicate.